modules/upgrade/pages/k-compatibility.adoc

```diff
@@ -0,0 +1,71 @@
+= Kubernetes Compatibility
+:description: Find a compatibility matrix for versions of Redpanda, Redpanda Helm chart, and Redpanda Operator.
+
+This topic describes the compatibility for versions of the Redpanda software, Redpanda Kubernetes Operator, and the Helm charts. The tables identify the versions required and supported for running the specified version of Redpanda.
+
+include::partial$versioning.adoc[]
+
+The latest version of the Redpanda Helm chart supports not only the most recent Redpanda release but also the previous two feature releases, subject to these considerations:
+
+- It supports the two preceding feature releases of the same year.
+- In cases where any of these two preceding feature releases are from the preceding year, the Helm chart supports the most recent feature release of that year.
+- Support is conditional on the preceding releases still being actively supported. In cases where any of the preceding releases are no longer supported, the Helm chart does not support them.
+
+== Redpanda Helm chart compatibility
+
+The Redpanda Helm chart is tested against these version of:
+
+- Redpanda software
+- Kubernetes and Helm
+
+[cols="4*m"]
+|===
+| Redpanda Helm Chart |Redpanda|Kubernetes|Helm
+
+| 5.6.x
+| 23.2.x, 23.1.x
+|1.27.3
+|3.11.1
+
+|===
+
+By default, the Redpanda Helm chart depends on cert-manager for enabling TLS.
+Make sure to install a version of the CRDs that supports the `certificates.cert-manager.io/v1` resource.
+See the https://github.com/cert-manager/cert-manager/releases[cert-manager release notes].
+
+== Redpanda Operator compatibility
+
+Each version of the Redpanda Operator Helm chart deploys a specific version of the Redpanda Operator software and the Redpanda Helm chart. The Redpanda Operator software is tied to the same version of the custom resource definitions (CRDs).
+
+The CRDs define the schemas used to configure Redpanda clusters. Each set of CRDs supports specific versions of the Redpanda Helm chart because they are used to translate the `clusterSpec` field of Redpanda resources to the values in the Redpanda Helm chart.
+
+Although new releases of the Redpanda Operator are backward compatible with existing versions of the Redpanda Helm chart, new releases of the Redpanda Helm chart might introduce changes to the Helm values, which could require you to upgrade both the Redpanda Operator and the accompanying CRDs.
+
+[cols="3*m"]
+|===
+| Redpanda Operator Helm Chart | Redpanda Operator | Redpanda Helm Chart
+
+| 0.4.x
```

Comment:
@mattschumpert we don't have access to previous chart versions. The idea was to start from this version and automate updating these tables on each new major/minor version: https://github.com/redpanda-data/helm-charts/blob/main/charts/operator/Chart.yaml

Reply:
Working on the automation which will fill in the table

Reply:
I need a way to determine which versions of the RP Helm chart each version of the RP Operator supports @RafalKorepta

Reply:
@mattschumpert We don't currently have a strategy to define the versions of the RP Helm chart that operator supports.

We should discuss this so that the docs can reflect our strategy.